flight_log_be/serializers.py

```diff
@@ -1,7 +1,27 @@
+# from rest_framework_json_api import serializers --> configuration statement if using JSON-API
 from rest_framework import serializers
+from rest_framework_json_api.relations import ResourceRelatedField
 from .models import User
+from .models import Flight
+
 
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ['id', 'first_name', 'last_name']
\ No newline at end of file
+        fields = ["id", "first_name", "last_name"]
```

Comment:
I took id out of my fields in the user serializer because I didn't want it to show up in the attributes. Did you get it to not show up?

Reply:
Nope, this was old - we removed it in the merge conflict!  I did figure out how to get it to now show up in Flights (since Brandon needs it to show up in his response) but it's very manual!